### ISSUE TITLE
Bookmarks and notes

### ISSUE THREAD

**Issue by sachinysawant** [logiclord/epubReader#24]
Hi Gaurav,
     This is the wonderful epubReader you developed for the ebook readers but i am getting notification as Please make a selection for note while adding notes and Please make a selection for bookmark while adding bookmark. So, i need your help for the same

**Comment by sureshkardam** [comment]
Dear Sachin !

i had this problem once and Gaurav told me to do this "You need to use selection mechanism of the reader. You need to activate the selection by hitting activation (space bar by default) then you will be able to selection using arrow keys.".....he is very nice and busy guy :)

**Comment by sachinysawant** [comment]
Yes it works as per your directions but from my point of view, hitting space bar and creating bookmarks and notes is not good user experience. So, if i want to remove this step (i.e hitting space bar) then which .js file i need to edit for the same

**Comment by sureshkardam** [comment]
you can follow the conversation on "https://github.com/logiclord/epubReader/issues/16#issuecomment-38936377"

**Comment by logiclord** [comment]
@suresh
Thanks suresh for helping out sachin.

@sachin
Yes, Suresh is right. Basically we need a way to notify reader that the
user is looking for selecting a specific section of the current page. You
change space bar to some other key as well.

**Comment by sachinysawant** [comment]
Thanks for your cooperation but i am not able to render the fixed layout epub even though there is no problem in the epub validated using http://validator.idpf.org as you referred in https://github.com/logiclord/epubReader/issues/16. For your information i am forwarding the screenshots and i am sending the same epub via email on Gaurav Aggarwal notifications@github.com
![epub_validator](https://cloud.githubusercontent.com/assets/9446873/4871526/dde191c8-61bc-11e4-8f23-584bd3fc11fe.png)

**Comment by sachinysawant** [comment]
Hi i am waiting for your response

**Comment by logiclord** [comment]
Apologies for delayed response. Yup this seems like a valid bug. I feel
this may be due to some parsing issues in table of content.

On Sun, Nov 2, 2014 at 10:30 PM, sachinysawant notifications@github.com
wrote:

> Hi i am waiting for your response
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/logiclord/epubReader/issues/24#issuecomment-61444115.

## 

Regards,

Gaurav Aggarwal
